app/api/dao/user.py

```diff
@@ -154,22 +156,26 @@ def list_users(user_id, is_verified=None):
         else:
             list_of_users = [user.json() for user in users_list]
 
+        for user in list_of_users:
+            relation = MentorshipRelationDAO.list_current_mentorship_relation(user['id'])
+            user['is_available_for_relation'] = False if isinstance(relation, MentorshipRelationModel) else True if user['need_mentoring'] or user['available_to_mentor'] else False
```

Comment:
Use the method and replace if else statement with boolean algebra `not or and`

Reply:
Sorry but I'm not able to get that, a little extra explanation would work a lot for me, if possible.

Reply:
I understood that, would implement that soon once I have access to my system.

Reply:
`user['is_available_for_relation'] = not userIsInRelation(user['id']]) and user['need_mentoring'] or user['available_to_mentor']` would that work?

Reply:
I think result of or should be in brackets

Reply:
I exactly implemented that (before seeing your comments)